### ISSUE TITLE
Doesn't works with `composer@1.7.0`

### ISSUE THREAD

**Issue by evilebottnawi** [wp-cli/wp-cli#4882]
Looks problem in composer https://github.com/composer/composer/issues/7516 (because with `1.6.5` all work good). But i can't have time to deep investigation, maybe something wrong in `composer.json` in this repo.

**Comment by schlessera** [comment]
Did you make sure it is not due to a stale Composer cache?

**Comment by evilebottnawi** [comment]
@schlessera try to remove cache, not solved problem

**Comment by schlessera** [comment]
The issue seems to be caused by the `wp-cli/autoload-splitter` we're using to get around package structure limitations in WP-CLI ~v1.

The files `vendor/composer/autoload_commands_classmap.php` and `vendor/composer/autoload_framework_classmap.php` contain only empty arrays.

**Comment by evilebottnawi** [comment]
@schlessera hm, but it is work as expected with 1.6.5 version, looks something changed in `composer`

**Comment by schlessera** [comment]
Yes, but they don't mention anything ni the changelog about the autoloader generation.

**Comment by wojsmol** [comment]
@schlessera Meyby comment on https://github.com/composer/composer/issues/7516 will be a good idea?

**Comment by schlessera** [comment]
@wojsmol The problem is already solved in commit https://github.com/composer/composer/commit/a74b63985e7cd811e5cdde30753249d8466666dd

The next update to Composer will fix this again.

**Comment by schlessera** [comment]
To get around this issue for now while we're waiting on a new version of Composer, you can pull the latest development snapshot of Composer like this:

```
composer self-update --no-plugins --snapshot
              ^            ^           ^
              |            |           +-- latest snapshot, instead of stable release
              |            +-------------- skip plugins as they are broken with v1.7.0
              +--------------------------- let Composer update itself
```
